evoks/evoks/middleware.py

```
@@ -1,25 +1,87 @@
         response = self.get_response(request)
         return response
 
-    def process_view(self, request, view_func, view_args, view_kwargs):
-        LOGIN_EXEMPT_URLS = ['logout', 'signup', 'login', 'reset_password/',
-                             'reset_password_sent/', 'reset_password_complete/', 'ToS']
+    @staticmethod
+    def basic_auth(request: HttpRequest):
+        if 'HTTP_AUTHORIZATION' in request.META:
+            auth = request.META['HTTP_AUTHORIZATION'].split()
+            if len(auth) == 2:
+                if auth[0].lower() == "basic":
```

Ruff (0.8.2)

Use a single `if` statement instead of nested `if` statements

Combine `if` statements using `and`

(SIM102)